cfbs.json

```diff
@@ -106,7 +106,7 @@
     },
     "inventory-lastlog": {
       "description": "Inventory kernel parameters set during system boot",
-      "tags": ["inventory", "experimental"],
+      "tags": ["experimental", "inventory"],
```

Comment:
I guess you meant this:

```suggestion
      "tags": ["supported", "inventory"],
```

Reply:
No, inventory-lastlog creates one attribute for each user who has logged in, since that's a dynamic number if could "blow up" mission portal

Reply:
Then don't change the order of the tags here.

Reply:
well, i had changed it to supported, then i changed it back.

Why is the order of tags important?